packages/fondue/src/components/RichTextEditor/Plugins/LinkPlugin/LinkButton.tsx

```diff
@@ -2,30 +2,26 @@
 
```

Comment:
`⌘+k` doesn't work but `Ctrl+k` does but the tooltip is wrong:

![Screenshot 2024-02-07 at 11 14 01](https://github.com/Frontify/fondue/assets/9362990/b62576b7-43f7-40e9-be2f-e1b9017edf14)

Reply:
Updated so Cmd + K works